### PR TITLE
README add extend-ops guide

### PR DESCRIPTION
**Pull request by stegben**
Add a simple guide on how to add a new op.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -53,3 +53,12 @@ result, meta = p.transform(['你好'])
...
 $ make test
 ```
+
+## Extend Ops
+
+1. Extend the new ops with `BaseOp`
+2. Define `input_type`, `output_type`
+3. Implement op creation
+4. Implement fit, transform, inverse_transform. If the op is stateless, the `fit` method should return None.
+5. write tests
```
Comment (absolutelyNoWarranty) [inline README.md]: 為什麼突然變小寫XD

Comment (stegben) [reply README.md]: fixed

File: README.md [hunk 2]
```diff
@@ -53,3 +53,12 @@ result, meta = p.transform(['你好'])
 ```
 $ make test
 ```
+
+## Extend Ops
+
+1. Extend the new ops with `BaseOp`
+2. Define `input_type`, `output_type`
+3. Implement op creation
+4. Implement fit, transform, inverse_transform. If the op is stateless, the `fit` method should return None.
```
Comment (absolutelyNoWarranty) [inline README.md]: "Note: It is expected that an ops's functionality will often be able to be decomposed into several functions. These functions should be written into (or imported from) the `toolkit` package for easy reuse.
`Ops` in the `ops` package will, for the most part, be wrappers for functions in `toolkit`.
"

Comment (stegben) [reply README.md]: added

**Pull request merged**